clustering/infinispan/extension/pom.xml

```diff
@@ -86,10 +104,6 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>wildfly-clustering-server-service</artifactId>
         </dependency>
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>wildfly-transactions</artifactId>
-        </dependency>
         <dependency>
             <groupId>org.wildfly.transaction</groupId>
             <artifactId>wildfly-transaction-client</artifactId>
```

Comment:
There is a -jakarta variant for wildfly-transaction-client, I guess we could use it here instead.

Reply:
@yersan Thanks. I picked that up in my v2 version of this -- #15871.